app/assets/stylesheets/utilities/_typography.scss

```diff
@@ -99,3 +99,8 @@ h6,
 .h6 {
   @extend %h6;
 }
+
+.text-center--line {
+  border-bottom-color: color('primary-light');
+  line-height: 0.1em;
```

Comment:
Is `0.1em` significant? Would it make any difference if it were `0` (collapse all line-height) or `1px` (match the size of the border)?